frontend/src/metabase-types/api/logger.ts

```diff
@@ -0,0 +1,36 @@
+export type LogLevel =
+  | "off"
+  | "fatal"
+  | "error"
+  | "warn"
+  | "info"
+  | "debug"
+  | "trace";
+
+export type TimeUnit =
+  | "days"
+  | "hours"
+  | "minutes"
+  | "seconds"
+  | "milliseconds"
```

Comment:
I'm not sure anything beyond seconds is useful

Reply:
@metamben FYI :arrow_up: 
Maybe it makes sense to remove it in backend as well.

----

Removed in frontend: 4ba70d12851d74f34b4e2384b5942fa22b45bc57

Reply:
It's useful on the backend for testing.